src/Commands/Images/GenerateSuppliers.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * 2019-present Friends of Presta community
```

Comment:
i think we should use the same licensing method than symfony, give a LICENSE file at the root of project and redirect every file to it.

Reply:
Not directly related with this pull, but feel free to open a new PR for the license management, this point is open :)